### DISCUSSION TITLE
Fatal error after updating to latest Version via Watchtower

### DISCUSSION THREAD

**Discussion by tamaskan** [photoprism/photoprism#2674] [Q&A]
`runtime: g 0: unknown pc 0x7f7ab3d2f83c

stack: frame={sp:0x7ffec5666fe0, fp:0x0} stack=[0x7ffec4e68560,0x7ffec5667570)

0x00007ffec5666ee0:  0x0000000000000000  0x0000000000000000 

0x00007ffec5666ef0:  0x0000000000000000  0x0000000000000000 

0x00007ffec5666f00:  0x0000000005343e90  0x00000000052c1140 

0x00007ffec5666f10:  0x00000000052ca7c0  0x00000000052c91c0 

0x00007ffec5666f20:  0x0000000000000000  0x0000000000000000 

0x00007ffec5666f30:  0x0000000000000000  0x0000000000000000 

0x00007ffec5666f40:  0x00000000052dc0d0  0x00000000052a6cf0 

0x00007ffec5666f50:  0x000000000532c820  0x0000000005345440 

0x00007ffec5666f60:  0x0000000000000000  0x0000000000000000 

0x00007ffec5666f70:  0x0000000000000000  0x0000000000000000 

0x00007ffec5666f80:  0x00000000052ac4d0  0x00000000052ad590 

0x00007ffec5666f90:  0x00000000052dbca0  0x00000000052d87d0 

0x00007ffec5666fa0:  0x0000000000000000  0x0000000000000000 

0x00007ffec5666fb0:  0x0000000000000000  0x0000000000000000 

0x00007ffec5666fc0:  0x6f6e206120736920  0x0a6649202e706f2d 

0x00007ffec5666fd0:  0x6572612079656874  0x00007f7ab3d2f82e 

0x00007ffec5666fe0: <0x0000000000000000  0x0000000000000000 

0x00007ffec5666ff0:  0x0000000000000000  0x0000000000000000 

0x00007ffec5667000:  0x0000000000000000  0x0000000000000000 

0x00007ffec5667010:  0x0000000000000000  0x0000000000000000 

0x00007ffec5667020:  0x0000000000000000  0x0000000000000000 

0x00007ffec5667030:  0x0000000000000000  0x0000000000000000 

0x00007ffec5667040:  0x0000000000000000  0x0000000000000000 

0x00007ffec5667050:  0x0000000000000000  0x0000000000000000 

0x00007ffec5667060:  0x0000000000000000  0xea5814f169686600 

0x00007ffec5667070:  0x00007f7ab1fe3900  0x0000000000000006 

0x00007ffec5667080:  0x00007ffec56673b0  0x0000000000000178 

0x00007ffec5667090:  0x0000000002dd7c00  0x00007f7ab3ce3a52 

0x00007ffec56670a0:  0x00007f7ab3e99e70  0x00007f7ab3cce469 

0x00007ffec56670b0:  0x0000000000000020  0x0000000000000000 

0x00007ffec56670c0:  0x0000000000000000  0x0000000000000000 

0x00007ffec56670d0:  0x0000000000000000  0x0000000000000000 

goroutine 1 [running]:

runtime.systemstack_switch()

	/usr/local/go/src/runtime/asm_amd64.s:459 fp=0xc000068780 sp=0xc000068778 pc=0x4720e0

runtime.main()

	/usr/local/go/src/runtime/proc.go:170 +0x6d fp=0xc0000687e0 sp=0xc000068780 pc=0x44566d

runtime.goexit()

	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000687e8 sp=0xc0000687e0 pc=0x474301

rax    0x0

rbx    0x7f7ab1fe3900

rcx    0x7f7ab3d2f83c

rdx    0x6

rdi    0x15

rsi    0x15

rbp    0x15

rsp    0x7ffec5666fe0

r8     0x7ffec56670b0

r9     0x7f7ab3e504e0

r10    0x8

`

**Comment by colemannugent** [comment]
I'm also running into what looks like the same issue:
```
started 220902 as root (amd64-prod)
Problems? Our Troubleshooting Checklists help you quickly diagnose and solve them:
https://docs.photoprism.app/getting-started/troubleshooting/
file umask....: "0002" (u=rwx,g=rwx,o=rx)
home directory: /photoprism
assets path...: /opt/photoprism/assets
storage path..: /photoprism/storage
config path...: default
cache path....: default
backup path...: /photoprism/storage/backups
import path...: /photoprism/import
originals path: /photoprism/originals
running as uid 0
/opt/photoprism/bin/photoprism start
runtime/cgo: pthread_create failed: Operation not permitted
SIGABRT: abort
PC=0x7f165c9aaa7c m=0 sigcode=18446744073709551610
```

I'm using the `latest` image from DockerHub. I can't think of a way this could be a user-induced issue.

@tamaskan Do you also get the `pthread_create failed: Operation not permitted` message?

@lastzero I can provide more info if you'd like to reopen #2673

**Reply by lastzero** [reply]
220902 looks more like our preview? It's Ubuntu based now. Also Go was upgraded. What CPU do you have? Might use features that old CPUs don't support.

**Reply by tamaskan** [reply]
Could be related to https://github.com/containers/skopeo/issues/1501

**Reply by lastzero** [reply]
Are you all using our configuration example at https://dl.photoprism.app/docker/docker-compose.yml? Have you modified the `security_opt:` section in any way?

**Reply by tamaskan** [reply]
it worked without "seccomp:unconfined" until the update, but i will set it if that's whats needed now

**Reply by lastzero** [reply]
Possible that the new Go version uses different kernel calls or there's other dependencies that do something different. Might need updated kernel security rules, which home users generally do not have experience with: https://docs.photoprism.app/getting-started/troubleshooting/docker/#kernel-security

**Reply by colemannugent** [reply]
Hmm upgrading Docker from `20.10.8` to `20.10.17` appears to have resolved the issue. No kernel changes, reboots, image changes, etc. besides that.

@tamaskan What version were you having trouble on? Does upgrading the Docker engine resolve the issue.

@lastzero It doesn't look like this is an issue with PhotoPrism. Thanks for the help!

**Reply by tamaskan** [reply]
the newest version that bumped the go-version. i changed the container to priviledged since i am running it via portainer and it works now
